File: src/Monad.AB.Domain.Entities/Project.cs
using System;
using System.Collections.Generic;

namespace Monad.AB.Domain.Entities
{
    public class Project : BaseEntity
    {
        public string UserName { get; set; }
        public string Name { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string RootNamespace { get; set; }
        public string CompanyName { get; set; }
        public DateTime CreatedDateUtc { get; set; }
        public DateTime LastModifiedDateUtc { get; set; }
        public int LastModifiedBy { get; set; }
        public IList<Form> Forms { get; set; }
        public bool AllowDownload { get; set; }
        public bool AllowPublish { get; set; }
        public DatabaseConfiguration DatabaseConfiguration { get; set; }
        public WebAppConfiguration WebServerConfiguration { get; set; }
    }
}

File: src/Monad.AB.Web.ng/src/app/projects/shared/project.ts
//shishir: need to evaluate if bases interface is nice to have
export class Project  {
    public id: number;
    public userName: string;
    public name: string;
    public title: string;
    public description: string;
    public rootNamespace: string;
    public companyName: string;
    public createdDateUtc: Date;
    public lastModifiedDateUtc: Date;
    public lastModifiedBy: number; 
    public allowDownload: boolean;
    public allowPublish: boolean;
}